executor_manager/services/sandbox/health_checker.py

```
@@ -0,0 +1,144 @@
+                            is_healthy = await self.check_health_async(port)
+                            logger.info(
+                                f"[ContainerHealthChecker] Health check for {container_name} "
+                                f"on port {port}: healthy={is_healthy}"
+                            )
+                            if is_healthy:
+                                logger.info(
+                                    f"[ContainerHealthChecker] Container ready: "
+                                    f"{container_name} on port {port}"
+                                )
+                                return port
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)